pkgs/applications/misc/skytemple/default.nix

```
@@ -2,13 +2,13 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "skytemple";
-  version = "1.3.2";
+  version = "1.3.9";
```

```suggestion
  version = "1.3.10";
```

This should actually have 1.3.10 (also, update checksum, obviously).